codeSnippets/snippets/full-stack-task-manager/composeApp/src/commonMain/kotlin/com/example/ktor/full_stack_task_manager/App.kt

```
@@ -0,0 +1,162 @@
+                    value = priorityText,
+                    onValueChange = { priorityText = it },
+                    label = { Text("Priority") },
+                    colors = colors
+                )
+                OutlinedButton(onClick = {
+                    val newTask = Task(
+                        task.name,
+                        description,
+                        try {
+                            Priority.valueOf(priorityText)
+                        } catch (e: IllegalArgumentException) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)